config/initializers/paperclip.rb

```diff
@@ -21,9 +21,9 @@
   Paperclip::Attachment.default_options[:use_timestamp]  = false
 
   Paperclip::Attachment.default_options[:s3_credentials] = {
-    bucket: ENV.fetch('S3_BUCKET'),
-    access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID'),
-    secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
+    bucket: ENV.fetch('BUCKETEER_BUCKET_NAME'),
```

Comment:
Hi, I'm sorry. This PR is my mistake. But I willing to work more better for this PR, if some one wants to run Mastodon on Heroku well.
Thanks.